grpc/execution/validation_test.go

```diff
@@ -32,6 +34,25 @@ func testDepositTx() *types.Transaction {
 	})
 }
 
+func generateBech32MAddress() string {
+	addressKey, err := crypto.GenerateKey()
+	if err != nil {
+		panic(err)
```

Comment:
instead of panicking, we can return an error and throw a `t.Fatal` where we use this method.

Reply:
I don't know why this will ever happen, copied how we behave elsewhere in the tests when generating a key